big_tests/tests/rdbms_SUITE.erl

```diff
@@ -72,7 +69,15 @@ suite() ->
 %% Init & teardown
 %%--------------------------------------------------------------------
 init_per_suite(Config) ->
-    escalus:init_per_suite(Config).
+    try not ct_helper:is_ct_running() orelse mongoose_helper:is_rdbms_enabled(host()) of
+        false -> {skip, bad_configuration};
+        true -> escalus:init_per_suite(Config)
+    catch
+        E:Err ->
+            ct:log("ERROR: ~p : ~p", [E, Err]),
+            {skip, init_error}
```

Comment:
Won't it result in the suite being skipped quietly if something is wrong? These functions should return valid booleans after all. Tests shouldn't pass if they crash for some reason.

Reply:
so true